### ISSUE TITLE
HA profile for Humidity sensor (TZ-78)

### ISSUE THREAD

**Issue by energizer91** [espressif/esp-zigbee-sdk#8]
I'm building custom thermometer using ESP32C6, SHTC3, Zigbee and esp-zigbee-sdk and i'm having some issues making it work.

For temperature sensor it's quite easy, you can just use `ESP_ZB_DEFAULT_TEMPERATURE_SENSOR_CONFIG()` and `esp_zb_temperature_sensor_ep_create`. You can also do that custom way by using `esp_zb_temperature_meas_cluster_add_attr`. But when it comes to Relative Humidity the only mention about it is in `esp_zigbee_zcl_common.h` and a bit in `esp-zboss-lib`. There's no way of adding humidity sensor using `esp-zigbee-sdk`. When I try to use `esp_zb_custom_cluster_add_custom_attr` it complains that cluster id is out of range, when i try `esp_zb_temperature_meas_cluster_add_attr` with Humidity sensor cluster id (0x0405) it complains that this id is not for temperature sensor.

So long story short: what's the way of adding humidity sensor using this SDK withouth rewriting project to some low-level implementation like `esp-zboss-lib`? Thanks!

**Comment by likunqiao097304** [comment]
@energizer91 Hi, the humidity sensor HA device, cluster and attribute support hasn't been implemented yet. So currently you can't make any humidity sensor product. We will add humidity support in the next round sdk release as need from your side.`esp_zb_custom_cluster_add_custom_attr` is for the user defined customized cluster or attribute that is not defined in the ZCL specs.
![image](https://user-images.githubusercontent.com/3393154/224209329-15d84d29-e18d-4f20-ac6d-14eb74f285f0.png)
